modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -567,14 +560,16 @@ use {{invokerPackage}}\ObjectSerializer;
         }
 
         // for model (json/xml)
-        if (isset($_tempBody)) {
-            // $_tempBody is the method argument, if present
+        {{#bodyParams}}
+        if (isset(${{paramName}})) {
             if ($headers['Content-Type'] === 'application/json') {
-                $httpBody = \GuzzleHttp\json_encode(ObjectSerializer::sanitizeForSerialization($_tempBody));
+                $httpBody = \GuzzleHttp\json_encode(ObjectSerializer::sanitizeForSerialization(${{paramName}}));
             } else {
-                $httpBody = $_tempBody;
+                $httpBody = ${{paramName}};
             }
-        } elseif (count($formParams) > 0) {
+        } else
```

Comment:
This abandoned `else` really bothers me... Did you forgot to delete it? Or maybe you wanted it as `elseif` as it was before?

Reply:
The issue is the `if` is only needed when you have the body, otherwise I delete it and the `elseif` becomes `if`. I don't know how to do it better with Mustache. 🙈

Reply:
Seems like I fixed it in my commit.